packages/components/popconfirm/__tests__/popconfirm.test.tsx

```diff
@@ -76,5 +77,27 @@ describe('Popconfirm.vue', () => {
       const { selector } = usePopperContainerId()
       expect(document.body.querySelector(selector.value)!.innerHTML).toBe('')
     })
+
+    it('should override default buttons when given actions', async () => {
+      const wrapper = mount(() => (
+        <>
+          <Popconfirm
+            v-slots={{
+              reference: () => <div class="reference">{AXIOM}</div>,
+              actions: () => <div class="actions">{FUN}</div>,
```

Comment:
IMO, It would be better to test whether `confirm` or `cancel` can work properly here. [refer to](https://github.com/element-plus/element-plus/blob/dev/packages/components/dialog/__tests__/dialog.test.tsx#L51)